generate_report.py

```diff
@@ -0,0 +1,41 @@
+import csv
+import sys
+
+from src.routes_report import RoutesReport
+from src.search_count_report import SearchCountReport
+
+def gen_report(filename):
```

Comment:
should this be `file_name` for consistency since you use `report_name` instead of `reportname` as a variable below

Reply:
isn't `filename` a word though?

Reply:
also prefer `generate_report`